plugins/core/impl/pom.xml

```diff
@@ -43,6 +43,13 @@
       </exclusions>
     </dependency>
 
+    <dependency>
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
```

Comment:
overrding the previous v 1.3 and using the v2.16.1 that is defined in the ancestor pom https://github.com/pentaho/maven-parent-poms/blob/f4b33561755f8159d8fad798d54c1506511b5250/pom.xml#L379